include/kernel/ebpf_registry_helper.h

```diff
@@ -13,123 +13,43 @@
 #define GUID_STRING_LENGTH 38 // not including the null terminator.
 
 typedef _Return_type_success_(NT_SUCCESS(return )) uint32_t ebpf_registry_result_t;
-
 typedef HANDLE ebpf_registry_key_t;
 
-static void
-close_registry_key(ebpf_registry_key_t key)
-{
-    ZwClose(key);
-}
-
-_Success_(return == 0) static NTSTATUS
-    convert_guid_to_string(_In_ const GUID* guid, _Out_writes_all_(string_size) wchar_t* string, size_t string_size)
-{
-    UNICODE_STRING unicode_string;
-    NTSTATUS status = RtlStringFromGUID(guid, &unicode_string);
-    if (status != STATUS_SUCCESS) {
-        goto Exit;
-    }
-
-    if (string_size < GUID_STRING_LENGTH + 1) {
-        status = STATUS_BUFFER_TOO_SMALL;
-        goto Exit;
-    }
-
-    __analysis_assume(unicode_string.MaximumLength >= GUID_STRING_LENGTH * 2);
-    __analysis_assume(unicode_string.Buffer != NULL);
-    // Copy the buffer to the output string.
-    memcpy(string, unicode_string.Buffer, GUID_STRING_LENGTH * 2);
-    string[GUID_STRING_LENGTH] = L'\0';
-
-Exit:
-    return status;
-}
-
-static _Must_inspect_result_ ebpf_registry_result_t
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Mocks for building platform_kernel & netebpfext when including the common ebpf_store_helper.h in kernel mode.
+// Currently these prototypes only need (existing) implementations in user mode, as they are not referenced in kernel
+// binaries.
+_Success_(return == 0) uint32_t open_registry_key(
```

Comment:
We might want to say ```(return == ERROR_SUCCESS)``` (like rather than use a hard-coded number)

Reply:
Genuine question: Why would including a header be that much of an issue?

Reply:
Even better, in this specific context It'd be more proper to use `(return == STATUS_SUCCESS)`, which does not even need a header -> queued this for the next commit.